pkg/models/dms.go

```diff
@@ -64,6 +64,7 @@ type EnrollmentSettings struct {
 	DeviceProvisionProfile      DeviceProvisionProfile      `json:"device_provisioning_profile"`
 	EnrollmentCA                string                      `json:"enrollment_ca"`
 	EnableReplaceableEnrollment bool                        `json:"enable_replaceable_enrollment"` //switch-like option that enables enrolling, already enrolled devices
+	EnableExpiredRenewal        bool                        `json:"enable_expired_renewal"`        // switch-like option that enables renewing expired certificates
```

Comment:
Since this a Certificate-specific property, place it in the `AuthOptionsClientCertificate` struct. On top of that, I would rename the prop into `AllowExpired`